go.sum

```diff
@@ -0,0 +1,136 @@
+github.com/bep/clocks v0.5.0 h1:hhvKVGLPQWRVsBP/UB7ErrHYIO42gINVbvqxvYTPVps=
```

Comment:
Ensure there's a consistent format for the dependency versioning, especially if you have both v2 and v3 modules mixed, which might confuse some users or tools.